### ISSUE TITLE
Display of an SVG at various places to show if user is banned or active

### ISSUE THREAD

**Issue by gauravano** [publiclab/plots2#3338]
Display if someone's been banned or is still active in moderator emails (https://github.com/publiclab/plots2/blob/master/app/views/admin_mailer/notify_node_moderators.html.erb) using the SVG at the above URL so moderators can tell if something's already been moderated without opening a link

SVG preferred: https://gist.github.com/jywarren/ea0523a09f6f8660486b14eeea15de95

**Comment by gauravano** [comment]
@jywarren, could you mention the flow how it can be useful like with an example. thanks!

**Comment by JonathanXu1** [comment]
That's a cool idea! On my localhosted project there's no emails to moderate:
![image](https://user-images.githubusercontent.com/22998430/49345247-3b72dc80-f650-11e8-8f08-99957b0f38b1.png)
If any of you have a moderator view of this page, a screenshot for design purposes would be appreciated. Alternatively, if there's a way to generate some on local host that would be nice to know too.

**Comment by gauravano** [comment]
You can generate test users and then play with the things. rails console
could help you fasten the user generation process or you can also create
the user by going through the normal process.

**Comment by gauravano** [comment]
Yes, because emails can only be sent in the production. But, I guess a
email tab is opened showing the email due to letter_opener ?

**Comment by JonathanXu1** [comment]
Oh okay. How can I open the email tab?

**Comment by gauravano** [comment]
It opens by default if mail is sent. Try creating a new user and see if new
tab(having email i text format) opens for you.

**Comment by JonathanXu1** [comment]
I've created a new account in a separate browser, and I still have no new tab opening up on my browser with the moderator logged in.

**Comment by oorjitchowdhary** [comment]
@SidharthBansal @gauravano, As @JonathanXu1 is focusing on other important issues.. I'm taking this up.. Hope you don't have a problem @JonathanXu1

**Comment by JonathanXu1** [comment]
Go ahead :)

**Comment by oorjitchowdhary** [comment]
Thanks a lot Jonathan.. onto it

**Comment by oorjitchowdhary** [comment]
@publiclab/reviewers Can someone help me with it's conditional..
It should be `if node.status == 4` ? or something else

**Comment by oorjitchowdhary** [comment]
@SidharthBansal I've some doubts for this.. I implemented the conditonal.. Now the email displays the active svg but I wonder how will we ever see the banned svg .. bcoz unless we mark a note spam, we will always see the active banner.. and once we spam a note, we also ban the user so he won't be able to publish any new notes and therefore we won't receive any email with the banned svg..
Please guide me here, if I'm going wrong

**Comment by SidharthBansal** [comment]
Jeff or Gaurav might help you here.
Good luck

**Comment by oorjitchowdhary** [comment]
@gauravano Please guide me here

**Comment by oorjitchowdhary** [comment]
I can send a PR.. Wanted to clarify this first.. But I'll submit the PR first then

**Comment by SidharthBansal** [comment]
You can only continue to work on those prs which will be sent before
today's deadline.
No PR will be reviewed of it will be sent after today's deadline.

**Comment by oorjitchowdhary** [comment]
Okay.. but I can claim my last task on gci which can be worked upon till Dec 12th .. right?

**Comment by oorjitchowdhary** [comment]
So what should I submit for my task.. @gauravano ?
@jywarren Help us here

**Comment by oorjitchowdhary** [comment]
So if user.status == 1, ACTIVE svg
else BANNED svg
Am I right?

**Comment by gauravano** [comment]
yes!

**Comment by oorjitchowdhary** [comment]
Okay.. PR coming up
